extensions/cryptography.py

```
@@ -0,0 +1,33 @@
+    # CLASS -> VARIABLES
+    hasher: argon2.PasswordHasher
+    # CLASS -> INIT
+    async def init(self, request: Request) -> namespace: 
+        self.hasher = argon2.PasswordHasher()
+        return self
+    # CLASS -> HASH
+    def hash(self, password: str) -> str:
+        return self.hasher.hash(password)
+    # CLASS -> VERIFY
+    def verify(self, code: str, password: str) -> bool:
+        try: return self.hasher.verify(code, password)
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)